src/lib/Select/properties.ts

```diff
@@ -6,4 +6,5 @@ export type SelectProperties = {
   selectedItemLabel: string | string[] | null;
   showSelectedItemInDropdown: boolean;
   selectMultipleItems: boolean;
+  dropDownIcon?: string | null;
```

Comment:
Do we need this `?` if using null?

I think we should have 2 different props one for `showDropDownIcon` one for dropdown icon, and a default icon should be provided if icon is not passed as prop.